app/controllers/events/view/tickets/orders/list.js

```diff
@@ -74,6 +74,27 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async resendConfirmation(order) {
+      let payload = {};
+      try {
+        payload = {
+          'data': {
+            'order' : order.identifier,
+            'user'  : this.authManager.currentUser.email
+          }
+        };
+        await this.loader.post('orders/resend-email', payload);
+        this.notify.success(this.l10n.t('Email confirmation has been sent to attendees successfully'));
+      } catch (error) {
+        console.log(error);
+        const errorMessage = error.message || error.errors[0].detail;
```

Comment:
What about the case where the rate limiting kicks in ? What is the error message sent by the server ? Does it come in here correctly ?

Reply:
@niranjan94 When even limit kicks in, it's handled by `error.message`. If it's other trivial error like sending confirmation for cancelled orders, it's handled by `error.errors[0].detail`. Otherwise the else condition

Reply:
@uds5501 for rate limiting the server would be sending a `429` if I'm right. When that comes, please show a proper and a more human-friendly error message

Reply:
Also, with translations